awscli/customizations/cloudformation/yamlhelper.py

```diff
@@ -10,7 +10,17 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
-import json
+import sys
+
+# OrderedDict was introduced in Python 2.7, if we're in 2.6 import
+# from the ordereddict library, and use simplejson to get object_pairs_hook.
+if sys.version_info[:2] == (2, 6):
```

Comment:
These import differences are already handled in the botocore compat module.
https://github.com/boto/botocore/blob/develop/botocore/compat.py#L144-L148
https://github.com/boto/botocore/blob/develop/botocore/compat.py#L151

```python
from botocore.compat import json
from botocore.compat import OrderedDict
```